app/models/exercise.rb

```diff
@@ -3,11 +3,6 @@ class Exercise < ApplicationRecord
   has_many :daily_exercise, through: :exercise_counts
 
   def self.empty_hash
-    exercises = all
-    result = {}
-    exercises.each do |exercise|
-      result[exercise.name] = 0
-    end
-    result
+    pluck(:name).zip([0].cycle).to_h
```

Comment:
oh god now this i am not so sure about!!!

Reply:
if u add a test i will accept this PR but otherwise i will reject it, not because i don't appreciate your input, more because your input is the best input i have ever seen and while i don't write tests i very much expect you to